File: Who has the most money.js
const mostMoney = (students) => {
  let save = []
  let max_v = 0
  for(let s of students){
    let tmp = s.fives*5 + s.tens*10 + s.twenties*20
    save.push([s.name,tmp])
    if(max_v < tmp){
      max_v = tmp
    }
  }
  let result = []
  for(let [n,v] of save){
    if(v===max_v){
      result.push(n)
    }
  }
  return result.length > 1 ? 'all' : result[0]
}

File: Who has the most money/Who has the most money.py
def most_money(students):
    result = {}
    for s in students:
        result[s.name] = s.twenties*20 + s.tens*10 + s.fives*5
    tmp = [k for k,v in result.items() if v==max(result.values())]
    return tmp[0] if len(tmp)==1 else 'all'
